axelor-talent/src/main/java/com/axelor/apps/talent/service/JobApplicationServiceImpl.java

```diff
@@ -93,14 +100,32 @@ protected Employee createEmployee(JobApplication jobApplication) {
           .getMainEmploymentContract()
           .setCompanyDepartment(jobApplication.getJobPosition().getCompanyDepartment());
     employee.setName(employee.getContactPartner().getName());
+    convertDmsFileToEmployeeFile(jobApplication, employee);
 
     return employee;
   }
 
+  protected void convertDmsFileToEmployeeFile(JobApplication jobApplication, Employee employee) {
+    List<DMSFile> dmsFileList =
+        dmsFileRepo
+            .all()
+            .filter(
+                "self.isDirectory = false AND self.relatedId = :relatedModelId AND self.relatedModel = :relatedModelName")
+            .bind("relatedModelId", jobApplication.getId())
+            .bind("relatedModelName", "com.axelor.apps.talent.db.JobApplication")
```

Comment:
please get the full class name dynamically (it will help if we rename the model)